com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/util/ApplicationResources.java

```diff
@@ -0,0 +1,28 @@
+package com.sap.cloud.lm.sl.cf.process.util;
+
+import java.util.Collection;
+
+import org.cloudfoundry.client.lib.CloudControllerClient;
+import org.cloudfoundry.client.lib.domain.CloudResource;
+import org.cloudfoundry.client.lib.domain.CloudResources;
+
+public class ApplicationResources {
```

Comment:
Isn't the object `CloudResources ` enough for this class??

Reply:
Fixed